src/config-schemas/openmrs-esm-patient-registration-schema.ts

```diff
@@ -35,4 +35,10 @@ export const esmPatientRegistrationSchema = {
       },
     ],
   },
+  registrationSubmission: {
+    afterCreateUrl: {
+      default: '${openmrsSpaBase}/patient/${patientUuid}/chart',
+      validators: [validators.isString],
```

Comment:
Please use the [isUrlWithTemplateParameters](https://github.com/openmrs/openmrs-esm-module-config/blob/fa8ba01b4718adfca46238e48276c10fc72ef782/src/validators/validators.ts#L38) validator. Sorry the validators aren't better-documented (these are the [docs](https://github.com/openmrs/openmrs-esm-core/tree/master/packages/esm-config#isurlwithtemplateparameters) right now, which are not very helpful, but are good for quickly seeing what validators exist, at least).

Reply:
Thanks for the pointers to the docs.

Reply:
Fixed